File: Dataset/Leetcode/test/26/50.java
 class Solution {
    public int XXX(int[] nums) {
        //返回的结果
        int res=nums.length;
        int i=0,j=1;
        while(j<nums.length){
            if(nums[i]==nums[j]){
                res--;
                j++;
            }else{
                nums[i+1]=nums[j];
                i++;j++;
            }
        }
        return res;
    }
}```

undefined
for (i = 0; i < document.getElementsByTagName("code").length; i++) {     console.log(document.getElementsByTagName("code")[i].innerText); }

File: Dataset/Leetcode/train/88/50.cpp
 class Solution {
public:
    void XXX(vector<int>& nums1, int m, vector<int>& nums2, int n) {
        nums1.erase(nums1.begin()+m,nums1.end());
        for(int i=0;i<n;i++)
        {
            nums1.push_back(nums2[i]);
        }
        sort(nums1.begin(),nums1.end());
    }
};

undefined
for (i = 0; i < document.getElementsByTagName("code").length; i++) {     console.log(document.getElementsByTagName("code")[i].innerText); }


